spec/tasks/gen/action_spec.cr

```diff
@@ -3,31 +3,35 @@ require "../../spec_helper"
 include CleanupHelper
 
 describe Gen::Action do
-  it "generates a basic action" do
+  it "generates a basic browser action" do
```

Comment:
I think it would be helpful to also have a test for basic (non-nested) API actions as well

Reply:
Sure, I'll duplicate the top test